src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -39,16 +39,12 @@ const useStyles = makeStyles((theme) => ({
 
 function CustomizedInputBase(props) {
   const classes = useStyles();
-  const [state, setState] = React.useState({
-    filter: '',
-  });
-  const { searchText, onChangeHandler } = props;
+  const [selectedFilter, setSelectedFilter] = useState(props.filter);
+  const { searchText, onChangeHandler, onFilterChangeHandler } = props;
```

Comment:
Same thing here.

Reply:
so don't use useState, and just use props? I removed all instances of assigning and states in `SearchBar.jsx` so now it looks like the below:

![image](https://user-images.githubusercontent.com/18711727/78934026-390c6800-7a78-11ea-9cea-6ce8882610c4.png)

Reply:
Red underlines don't tell me anything.  What is it complaining about when you hover?  Are you not using these values anywhere?

Reply:
Nono, I didn't mean it like it was erroring. I mean after your requested changes it should look like this without any `useState` right?